src/payment/strategies/googlepay/googlepay-payment-processor.ts

```diff
@@ -217,6 +217,18 @@ export default class GooglePayPaymentProcessor {
         }
 
         const googlePayAddressMapped = this._mapGooglePayAddressToBillingAddress(paymentData, remoteBillingAddress.id);
+        const firstName = 'firstName';
```

Comment:
I will rather suggest to move this block to move this code to `_mapGooglePayAddressToBillingAddress` function.

Please include a tes case for such scenario too.

Reply:
ok